client/dashboard/sites/settings-mcp/style.scss

```diff
@@ -0,0 +1,318 @@
+.mcp {
+	&__type-section {
```

Comment:
Do we need these styles? We should follow the core styles instead of customizing lots of things. Also, we don't use color schemes anymore on v2 so we should not have css variables like `--color-neutral-10`